src/extension/iSamples_hierarchyFacet.js

```diff
@@ -101,7 +101,7 @@ class HierarchyFacet extends React.Component{
         const expanded = !(collapse || false);
 
         return (
-        <li className={cx("hierarchy-facet", { "hierarchy-group-item": bootstrapCss })} id={`solr-hierarchy-facet-${field}`}>
+        <li className={cx("hierarchy-facet", { "list-group-item": bootstrapCss })} id={`solr-hierarchy-facet-${field}`}>
```

Comment:
As we do not have defined css for hierarchy-facet, we just use the list-facet 's css for rendering. I think this would be a better approach then altering/redefining a css property for hierarchy-facet.